content/asp.net/getting-started/results.md

```diff
@@ -1,20 +1,88 @@
-# Returning Data or Views from Controllers
-by [Steve Smith](http://deviq.com/me/steve-smith)
+# Understanding Controllers and Actions
+by [James Chambers](http://jameschambers.com)
 
 #### Sample Files
 Download a ZIP containing this tutorial's sample files:
-- [Initial Version] - Use this as a starting point when following along with the tutorial yourself
-- [Completed Version] - Includes the completed versions of all samples
+- [Completed Version (Pending)] - Includes the completed versions of all MVC tutorials
 
-See the [Issue](https://github.com/dotnet/training-tutorials/issues/61) to claim this lesson and/or view what should be included in it.
+## Returning Views in Your App
+We can have an action in `PersonController` return a view with the following code:
+```c#
+public IActionResult Index()
+{
+    return View();
+}
+``` 
 
-## First Header
+As [previously discussed](controllers-actions.md), the call to `View()` signals the framework to continue processing the request, and it will do so with a default convention. It assumes that the name of the action will have a corresponding view at `Views/Person/Index.cshtml`. 
```

Comment:
Not sure how/where to add it, but it will also check in "/Views/Shared/[viewname].cshtml" as well, which works for both "action name as convention" and "explicit viewname string parameter" mechanisms of specifying the view name.